src/Nethermind/Nethermind.Consensus/ExecutionRequests/ExecutionRequestProcessor.cs

```diff
@@ -131,25 +133,37 @@ private void ReadRequests(Block block, IWorldState state, IReleaseSpec spec, Add
 
         int requestsByteSize = isWithdrawalRequests ? ExecutionRequestExtensions.WithdrawalRequestsBytesSize : ExecutionRequestExtensions.ConsolidationRequestsBytesSize;
 
-        if (!(isWithdrawalRequests ? spec.WithdrawalRequestsEnabled : spec.ConsolidationRequestsEnabled) || !state.AccountExists(contractAddress))
+        if (!(isWithdrawalRequests ? spec.WithdrawalRequestsEnabled : spec.ConsolidationRequestsEnabled))
```

Comment:
can we remove `isWithdrawalRequests` and change all those `if's` to come from method parameters?

Reply:
Nice idea